src/modules/options/lineListConfig.js

```diff
@@ -17,5 +17,9 @@ export default () => [
             ]),
         },
     ]),
-    getLegendTab(),
+    ...(`${serverVersion.major}.${serverVersion.minor}.${
+        serverVersion.patch || 0
+    }` > '2.38.1'
```

Comment:
This doesn't work in some cases, for example `'12.0.0' > '2.38.1'`, probably not a problem in the short term, but if we ever decide to skip to a major `>=10` we need to change the comparison.